### DISCUSSION TITLE
2023 Maps for ROW or Region Map Change from ROW to Europe

### DISCUSSION THREAD

**Discussion by MASMahmood** [Mr-MIBonk/M.I.B._More-Incredible-Bash#138] [General]
Hi Guys,

I know there will be a guide for this some where but I am unable to locate it - feel free to post link to it.

My friend has an 2016 Audi S3 Saloon imported from Australia (to the UK) he recently paid someone to update change his region to UK (via M.I.B).

I checked his maps and it states ROW (Rest Of World) also his maps are 2017 which I pointed out to him, I did advise him to speak to the individual to push 2023 maps (if there are available) on to his MMI via SD Card,

His version is MHI2_ER_AU37X_P3620.

I have a 2017 Audi S5 Sportback was bought from Audi UK dealer and my maps are Europe (ECE), I recently patched my system (MHI2Q_ER_AUG22_P3638) added FECs (new maps) and updated my maps to 2023 using the maps from this website:

https://app-connect.volkswagen.com/mapupdates/en/download/#download

But I am unable to find maps for ROW.

Is there a link to download the latest ROW maps?

Or would my friend have to get his region changed to ECE and download the latest maps?

What is the process to change region and install the maps? I assume he has to delete his current maps installed.

If anyone could provide some instructions on how to do this thanks.

**Comment by harman-f** [comment]
Somebody did a great job!

Why do you ask for ROW maps? From what you wrote the car is now in UK --> EU

Use M.I.B in GEM to:
- code navigation to EU
- format NavDB
- make sure unit is patched
- Add EU Map FEC - addFEC function
- Install EU maps
- You also might want to check coding for SDS, radio if it was changed to EU as well.

**Reply by MASMahmood** [reply]
@harman-f I told him once I did my own maps he got ripped off!

I have no idea why the guy installed ROW map and not EU maps? (would you believe it was teenager who he trusted his expensive car to do this region change with)

Correct the car in the UK now.

I am newbie (idiot) to stuff like this 

I was thinking if I get the ROW maps I can just be lazy and update from 2017 to 2023.

Ok seems straight forward I thought he may have to do something with firmware seems like it is not the case.

On this step:

make sure unit is patched - doesn't this step also cover the EU map FEC?

I did check this activation keys he does have 023000EE which is the Audi EU Maps - Lifetime.

His Activations keys are:

00060800
00030000
00070200
00050000
023d0022
00040100
023000ee

On the last step I might just leave for him as I don't have clue what SDS is? I will ask him to check his radio and see if it picks up UK radio stations.

**Reply by harman-f** [reply]
Looks like patch was applied in that case.

I think coding for SDS is also changed with NAV region change in GEM.

Coding for Radio can also be checked in GEM.

**Reply by MASMahmood** [reply]
@harman-f thanks for the info.

Will advise how it gets on.

**Comment by aledepaoligampel** [comment]
How can i install ROW maps with EU firmware?
I try everything, changing in MIB:
- SDS and Navigation country to MSA
- In car variation from Europe to RoW (and Europe/RoW too)

My FEC's are:
021D00EE
021000EE

When I install the EU maps, works perfect. but when install the RoW maps, the message in nav is: the navigation data are invalid.
I try to format nav_db too.

Can anybody helpme. I try everything for weeks, installing, formatting and nothing works.
My car is Audi A3 MIB1High MHIG

**Comment by charitos151** [comment]
Hello,

Same kind of issue here. I have an Audi A5/F5 imported from Australia to Europe(Cyprus), my radio firmware is MHI3_ER_AU_P3853_8W0906961BA but the radio firmware is on the Rest of World. How I can change it to Europe?
Can I do it throw OBD?
Thanks

**Comment by Vittorio78** [comment]
Ciao, qualcuno potrebbe inviarmi l'attivatore MTSD da un appassionato perché il MIBwiki.one è morto. thx à lot
